.sass-lint.yml

```diff
@@ -19,7 +19,7 @@ rules:
   # Disallows
   no-color-keywords: 1
   no-color-literals: 1
-  no-css-comments: 1
+  no-css-comments: 0
```

Comment:
Why do we allow css comments? Or is it to be used for the generated template css stylesheet?